buzzer/client.ts

```diff
@@ -47,7 +47,8 @@ namespace modules {
                 startPlay = true
             }
             for (let off = 0; off < buf.length - (isize - 1); off += isize) {
-                const [soundWave, flags, frequency, duration, startVolume, endVolume, endFrequency] = buf.unpack("BBHHHHH", off)
+                const [soundWave, flags, frequency, duration, startVolume, endVolume, endFrequency] =
+                    jacdac.jdunpack<[number, number, number, number, number, number, number]>(buf, "u8 u8 u16 u16 u16 u16 u16", off)
```

Comment:
should we have a jdunpackNumbers or something that specializes the return type to `number[]` ?

Reply:
using number[] now